packages/storybook/stories/snippets/transfer/Transfer.snippet.tsx

```diff
@@ -0,0 +1,81 @@
+import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
+import {
+  CallAdapter,
+  CallComposite,
+  CallCompositeOptions,
+  CompositeLocale,
+  useAzureCommunicationCallAdapter
+} from '@azure/communication-react';
+import { PartialTheme, Theme } from '@fluentui/react';
+import React, { useCallback, useMemo } from 'react';
+
+export type ContainerProps = {
+  userId: CommunicationUserIdentifier;
+  token: string;
+  formFactor?: 'desktop' | 'mobile';
+  fluentTheme?: PartialTheme | Theme;
+  locale?: CompositeLocale;
+  options?: CallCompositeOptions;
+  microsoftTeamsUserId?: string;
+};
+
+export const ContosoCallContainer = (props: ContainerProps): JSX.Element => {
+  const credential = useMemo(() => {
+    try {
+      return new AzureCommunicationTokenCredential(props.token); // <-- This props.token would be your Teams access token
+    } catch {
+      console.error('Failed to construct token credential');
+      return undefined;
+    }
+  }, [props.token]);
+
+  const callAdapterArgs = useMemo(
+    () => ({
+      userId: props.userId,
+      credential,
+      locator: props.microsoftTeamsUserId
+        ? {
+            participantIds: [props.microsoftTeamsUserId]
+          }
+        : undefined
+    }),
+    [props.userId, credential, props.microsoftTeamsUserId]
+  );
+
+  const afterCallAdapterCreate = useCallback(async (adapter: CallAdapter): Promise<CallAdapter> => {
+    adapter.on('transferRequested', (e) => {
```

Comment:
can we change the name from `e` for the event param to `transfer` or something indicative of the object in the event we are actioning `accept` on?